changelog.txt

```diff
@@ -1,5 +1,8 @@
 0.83.1
  * Added Realm.isClosed() method.
+ * Now IllegalStateException is thrown instead of native crash if following methods in RealmList are called after closing Realm or removing parent object: add(int,RealmObject), add(RealmObject),
+ * Now IllegalStateException is thrown instead of ArrayIndexOutOfBoundsException if following methods in RealmList are called after closing Realm or removing parent object: set(int,RealmObject), move(int,int), remove(int), get(int)
```

Comment:
Now IllegalState is -> IllegalStateException is now
"after closing Realm" -> "after closing the Realm"